packages/advanced-logic/test/utils/payment-network/erc777/stream-create-data-generator.ts

```diff
@@ -197,6 +234,48 @@ export const requestFullStateCreated: RequestLogicTypes.IRequest = {
   version: '0.1.0',
 };
 
+export const subsequentRequestFullStateCreated: RequestLogicTypes.IRequest = {
+  creator: {
+    type: IdentityTypes.TYPE.ETHEREUM_ADDRESS,
+    value: TestData.payeeRaw.address,
+  },
+  currency: {
+    network: 'rinkeby',
+    type: RequestLogicTypes.CURRENCY.ERC777,
+    value: '0x745861aed1eee363b4aaa5f1994be40b1e05ff90', //fDAIx
+  },
```

Comment:
Another example here, creating an fDAIx constant with this value avoids the quadruple `//fDAIx` definition & comment.